src/catkin_pkg/package.py

```diff
@@ -696,7 +696,25 @@ def parse_package_string(data, filename=None, warnings=None):
         })
     nodes = [n for n in root.childNodes if n.nodeType == n.ELEMENT_NODE]
     unknown_tags = set([n.tagName for n in nodes if n.tagName not in known.keys()])
-    if unknown_tags:
+    if unknown_tags == set(['run_depend']) and pkg.package_format >= 2:
+        errors.append('\033[91mWARNING:')
+        errors.append('WARNING: ACTION REQUIRED')
+        errors.append('WARNING:')
+        errors.append('WARNING: The manifest of package "%s" (with format version %d) '
+                      'does not support <run_depend>. ' % (pkg.name, pkg.package_format))
```

Comment:
Please remove the trailing space in the message. It is not necessary since each item is printed on a separate line anyway.